satpy/etc/readers/eps_sterna_mwr_l1b_nc.yaml

```
@@ -0,0 +1,532 @@
+reader:
+  name: eps_sterna_mwr_l1b_nc
+  short_name: AWS L1B RAD NetCDF4
+  long_name: AWS L1B Radiance (NetCDF4)
+  description: Reader for the EUMETSAT EPS-Sterna Sounder level-1b files in netCDF4.
```

I woudn't call MWR a sounder.
```suggestion
  description: Reader for the EUMETSAT EPS-Sterna radiometer level-1b files in netCDF4.
```